web/modules/custom/speech/speech.routing.yml

```diff
@@ -0,0 +1,12 @@
+speech.call:
+  path: '/conference-call'
+  defaults:
+    _controller: '\Drupal\speech\Controller\ConferenceCallController::content'
```

Comment:
Please call a form builder here, instead of calling a controller that loads the form.